### DISCUSSION TITLE
Can ZAPP affect the smartphone alarm clock?

### DISCUSSION THREAD

**Discussion by WebworkrNet** [mediathekview/zapp#436] [Q&A]
Samsung Galaxy A13 5G (SM-A136B)
Android 14, One UI 6.0

I occasionally notice that when using ZAPP (possibly in sleep mode), the smartphone's system alarm clock may not be set.

Indications:

- The alarm clock does not ring.
- Missing PUSH notification about missed alarm.

Could ZAPP be the cause?
Thanks

**Comment by cemrich** [comment]
This is highly unlikely. The only thing messing with time in Zapp is the sleep timer, and it uses a simple kotlin handler - no system calls here.
